### ISSUE TITLE
Support Snowflake DB

### ISSUE THREAD

**Issue by nalvest** [HeidiSQL/HeidiSQL#1091]
Enhancement request to add support for accessing Snowflake DB.

https://docs.snowflake.com/en/user-guide/conns-drivers.html

It's SQL based so should be easier to support than say Mongo or another NoSQL db.

**Comment by ansgarbecker** [comment]
HeidiSQL normally needs a C-API as driver. Snowflake does not provide such a library, only an ODBC driver, which HeidiSQL would then have to implement. Or is there some C-API somewhere else?
And who the heck uses Snowflake? ;)

**Comment by nalvest** [comment]
Yes, there isn't a C-API driver available.  I just checked with them and there are no plans at the current time.

Snowflake is used by large companies for data warehouse / data lakes. It's become extremely popular in that space in the last few years and is comparable to Google BigQuery and Amazon RedShift.  I consider it to be a better solution than both of those for various reasons that this isn't a good forum for :)

**Comment by boxi360** [comment]
Good Morning, any news regarding Snowflake support ?

**Comment by lukinhaspm** [comment]
@ansgarbecker is possible to do? I work in a very vig company who use a lot Snowflake and cam use Heidi as well

**Comment by ansgarbecker** [comment]
Very unlikely. I already have 4 incomplete implementions for MS SQL, Postgres, SQLite and Interbase. I should not add further drivers without the capabilities to support them.
